content/en/tracing/visualization/service.md

```diff
@@ -41,69 +44,135 @@ Enable them directly or create your own [APM monitors][3].
+* **Error Rate**: Percent of error for a given resource
 
-[Refer to the dedicated resource documentation to learn more][2].
+{{< img src="tracing/visualization/service/resource_columns.png" alt="Resource columns"  style="width:40%;">}}
 
-### Filtering the resources list
+## Additional Tabs
 
-Filter your resources list with a query for basic text filtering:
+### Deployments
+A service configured with version tags will show versions in the Deployment tab. The version section shows all versions of the service that were active during the selected time interval, with active versions at the top.
 
-{{< img src="tracing/visualization/service/resources_filtering.mp4" alt="Resource filtering" video="true"  width="90%" >}}
+By default you will see:
+* The version names deployed for this service over the timeframe.
+* The times at which traces that correspond to this version were first and last seen.
+* An Error Types indicator, which shows how many types of errors appear in each version that did not appear in the immediately previous version.
 
-### Columns
+    **Note**: This indicator shows errors that were not seen in traces from the previous version. It doesn’t mean that this version necessarily introduced these errors. Looking into new error types can be a great way to begin investigating errors.
 
-Choose what to display in your resources list:
+* Requests per second.
+* Error rate as a percentage of total requests.
 
-* **Requests**: Absolute amount of requests traced (per seconds)
-* **Avg/p75/p90/p95/p99/Max Latency**: The Avg/p75/p90/p95/p99/Max latency of your traced requests
-* **Total time**: Sum of all time spend in this resource
-* **Error**: Absolute amount of error for a given resource
-* **Error Rate**: Percent of error for a given resource
+You can add columns to or remove columns from this overview table and your selections will be saved. The additional available columns are:
+
+* Endpoints that are active in a version that were not in the previous version.
+* Time active, showing the length of time from the first trace to the last trace sent to Datadog for that version.
+* Total number of Requests.
+* Total number of Errors.
+* Latency measured by p50, p75, p90, p95, p99, or max.
+
+{{< img src="tracing/visualization/service/deployments.png" alt="Deployments"  style="width:90%;">}}
+
+Read more about Deployments on the service page [here][12].
+
+### Error Tracking
+View issues on your service, which are similar errors aggregated together to turn a noisy stream of errors into manageable issues, and help you assess the impact of your service’s errors. Read more about issues in [Error Tracking][9])
+
+This tab has overview graphs that show which resources have the most issues, and a list of the most common issues occurring in your service. Click on an issue in the list to see details in a side panel, including its stack trace, related code versions, and total error occurrences since inception.
```

Comment:
```suggestion
This tab has overview graphs that show which resources have the most issues and a list of the most common issues occurring in your service. Click on an issue in the list to see details in a side panel, including its stack trace, related code versions, and total error occurrences since inception.
```